### ISSUE TITLE
Symbol name does not match geometry

### ISSUE THREAD

**Issue by csmoore** [Esri/joint-military-symbology-xml#260]
While doing the Military Overlay for 2525C, I noticed these names still appear incorrectly mapped to their geometry type and this is confusing to users. I realize these are the Entity names directly used in 2525D, but I wanted to:

1. confirm that a CP had been written to try to address this (and get number).
2. record this as a known issue in this repo

This is not an exhaustive list, just the ones I noticed in a quick scan while doing the overlay:

##### In "Points" FeatureClass/Domain but "Areas" in name:

Fires Areas : Fire Support Station,G-F-PCF---
Protection Areas : Minefield : Dummy,G-G-PN----
Fires Areas : Point Targets : Nuclear Target,G-F-PTN---
Fires Areas : Point Targets : Point or Single Target,G-F-PTS---

##### In "Lines" FeatureClass/Domain but "Points","Areas" in name:

Fires Areas : Linear Targets : Linear Target,G-F-LT----
Fires Areas : Linear Targets : Final Protective Fire (FPF),G-F-LTF---
Fires Areas : Linear Targets : Linear Smoke Target,G-F-LTS---
Maneuver Areas : Search Area/Reconnaissance Area,G-G-GAS---
Maneuver Areas : Attack by Fire Position,G-G-OAF---
Maneuver Areas : Support by Fire,G-G-OAS---
Maneuver Areas : Axis of Advance : Main Attack,G-G-OLAGM-
Maneuver Areas : Axis of Advance : Supporting Attack,G-G-OLAGS-
Maneuver Areas : Axis of Advance : Attack Helicopter,G-G-OLAR--
Maneuver Areas : Axis of Advance : Friendly Airborne/Aviation,G-G-OLAV--
Maneuver Areas : Axis of Advance : Feint,G-G-PA----
Protection Areas : Bridge,G-M-BCB---
Protection Areas : Ford Difficult,G-M-BCD---
Protection Areas : Ford Easy,G-M-BCE---
Protection Areas : Obstacle Bypass : Difficult,G-M-BDD---
Protection Areas : Obstacle Bypass : Easy,G-M-BDE---
Protection Areas : Obstacle Bypass : Impossible,G-M-BDI---
Protection Areas : Obstacle Effects : Block,G-M-OEB---
Protection Areas : Obstacle Effects : Disrupt,G-M-OED---
Protection Areas : Obstacle Effects : Fix,G-M-OEF---
Protection Areas : Obstacle Effects : Turn,G-M-OET---
Protection Areas : Bridge or Gap,G-M-OFG---
Protection Points : Vertical Obstructions : Overhead Wire,G-M-OHO---
Protection Points : Abatis,G-M-OS----
Maritime Points : Navigational,G-O-HN----
Maneuver Areas : Battle Position : Contain,G-T-J-----
Maneuver Areas : Battle Position : Retain,G-T-Q-----

##### In "Areas" FeatureClass/Domain but "Lines" in name:

Maneuver Lines : Airhead Line,G-G-SAA---

This issue was captured in 2014 here - but a CP number did not get recorded with that issue: https://github.com/ArcGIS/military-features-pro-data_RETIRED_DONT_USE/issues/17 (Internal Link)

**Comment by abouffard** [comment]
Consulted SSMC 16-1 - decision is Joe will review.

**Comment by csmoore** [comment]
@abouffard @joebayles - does that mean review and then submit a CP with a complete list? Or has a CP previously been submitted for this issue?

**Comment by joebayles** [comment]
@csmoore the response was that most of these name disparities come from a doctrinal (vs. engineering) standpoint. For instance, Fire Support Station is more related to Fires Area control measures instead of other Fires Point control measures. So, @abouffard's comment means that I will review the list and more clearly differentiate between those that were named that way because of doctrine vs. those that might have been named incorrectly. From there we can re-assess what needs to be done.

**Comment by csmoore** [comment]
Thanks for the clarification - I would just make the comment that every time I (and others, like when Ben wrote this up as a defect in the old repo) see these names/labels in Pro, we think it is bug until/unless we remember this peculiar issue with the entity names.

Using the `Entity : Type : SubType` name format seems to work great everywhere else except for this one case so it would be good if this could be improved even to just add a clarification in the entity name like "Fire Areas (Fire Points)" - this is how a user sees now: 

![image](https://cloud.githubusercontent.com/assets/3090809/13881953/8a000fb6-ecf9-11e5-8d28-1f301dbc966c.png)

**Comment by joebayles** [comment]
Agreed it's unclear. Committee's argument was that it would make sense to those people that understand FSCM (Fires Support Control Measures). I'll go through the list and from there we can come up with a course of action re: how to amend the standard.

**Comment by joebayles** [comment]
While some of these have changed from 2525B Change 2 to 2525D and 2525D Change 1, the issues continue. The trends I see are:

- Areas that have point draw rules (minefields)
- Geometries that are listed together because they are related, but break taxonomy (fires, obstacles, etc.)

I've given this list to the SSMC, and they will investigate. Easy fix would be to adjust the hierarchical names (i.e. changing Fires Areas to Fires Control Measures, which would generalize the top level hierarchy).

FYI @mepler @ottenw

**Comment by mepler** [comment]
Unfortunately, the "easy" fix is not an easy fix. Example: Looking at Fires Areas Entity (240000) that has some line and point Entity Types under it, it would seems to make sense to just change the Entity name until you look at the following Entity (Fire Points - 250000). This is an issue that the SSMC needs to review more thoroughly. The last option we want is to change the code itself in order to place the Points and Lines so they are under the proper Entity.

**Comment by joebayles** [comment]
Discussed at 17-3. Could change listing earlier than 2525E in a User/Implementer Guide, but can't change the code.
